lib/containers/utils.pm

```diff
@@ -25,7 +25,11 @@ use version_utils;
 use Mojo::Util 'trim';
 
 our @EXPORT = qw(test_seccomp basic_container_tests get_vars can_build_sle_base check_docker_firewall
-  get_docker_version check_runtime_version container_ip registry_url);
+  get_docker_version check_runtime_version container_ip registry_url is_container_test);
+
+sub is_container_test {
+    return (check_var('EXTRATEST', 'cont'));
```

Comment:
Not sure if `cont` is already used, but I would vote for `container` as `cont` could also be `continuous`, `continental`

Reply:
Also, maybe it is a good idea to move the `is_container_test` to `lib/main_containers.pm`